core/src/main/scala/kafka/admin/TopicCommand.scala

```diff
@@ -536,6 +548,8 @@ object TopicCommand extends Logging {
                                                             "if set when describing topics, only show under replicated partitions")
     private val reportUnavailablePartitionsOpt = parser.accepts("unavailable-partitions",
                                                             "if set when describing topics, only show partitions whose leader is not available")
+    private val reportUnderMinIsrPartitionsOpt = parser.accepts("under-minisr-partitions",
```

Comment:
In the KIP the config is proposed as `under-min-isr-partitions`. Is there a reason you used a different name? I think the name in the KIP (with a dash between `min` and `isr`) looks and reads better.

Reply:
Good catch, thanks!